upbound_uxp/metadata.csv

```diff
@@ -0,0 +1,46 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+uxp.certwatcher_read_certificate_errors_total,count
```

Comment:
Looks like only the first two columns are filled out for each row here. 
Filling these out fully provides a better user experience for anyone using your integration. 
(Ex: Descriptions will show up in the Metrics Summary Page + the tile so users will know exactly what a given metric means; and `unit` information will make graphing easier)

Reply:
Updated.